x-pack/plugins/stack_connectors/public/connector_types/slack_api/translations.ts

```diff
@@ -43,7 +43,7 @@ export const ACTION_TYPE_TITLE = i18n.translate(
 export const ALLOWED_CHANNELS = i18n.translate(
   'xpack.stackConnectors.components.slack_api.allowedChannelsLabel',
   {
-    defaultMessage: 'Channels',
+    defaultMessage: 'Channel Ids',
```

Comment:
```suggestion
    defaultMessage: 'Channel IDs',
```

Alternatively, "Channel identifiers"